lib/katello/tasks/import_yum_metadata_file.rake

```diff
@@ -0,0 +1,11 @@
+namespace :katello do
+  task :import_yum_metadata => ["environment"] do
+    User.current = User.anonymous_api_admin
+    puts _("Importing Yum Metadata Files")
+    Katello::YumMetadataFile.import_all
+    Katello::Repository.linked_repositories.each do |repo|
+      base_repo = repo.target_repository
+      Katello::YumMetadataFile.copy_repository_associations(base_repo, repo)
+    end
+  end
+end
```

Comment:
any reason to not put this under the upgrades directory?  This should also be added to the upgrades seed file.

Reply:
addressed.